src/oop/programs_with_interesting_classes/medium/q1/QuestionText.md

```diff
@@ -0,0 +1,63 @@
+# oop-programs_with_interesting_classes-medium-q1
+
+- Parsons Question
+
```

Comment:
Wouldn't this be better as java I think its to long to be a parson

Reply:
I think it is better to have it as a parsons. It would still be the same length as a java question.

Reply:
We have not done many parsons questions for objects, so I think it is good to have a category that has some more.